File: app/src/main/java/com/nevbg/simple/activity/MainActivity.java
package com.nevbg.simple.activity;

import android.app.Activity;
import android.os.Bundle;
import android.support.annotation.Nullable;

import cn.sft.util.Util;

/**
 * APP主界面
 * 在WelcomeActivity中返回MainActivity.class进行关联绑定
 */
public class MainActivity extends Activity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        Util.print("MainActivity");
        super.onCreate(savedInstanceState);
    }
}

File: app/src/main/java/com/nevbg/simple/activity/MainActivity.kt
package com.nevbg.simple.activity

import android.os.Bundle
import com.nevbg.simple.R
import com.nevbg.simple.base.CustomBaseActivity
import com.wxt.library.util.Util

/**
 * APP主界面
 * 在WelcomeActivity中返回MainActivity.class进行关联绑定
 */
class MainActivity : CustomBaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Util.print("MainActivity")
    }

}